src/components/ui/ControlBar/ControlBarItem.tsx

```diff
@@ -44,16 +42,22 @@ export const ControlBarButton: FC<ControlBarButtonProps> = ({
   popOver = null,
   popOverPlacement,
   popOverLabel,
+  tooltipContainerId,
+  tooltipText,
   children,
   ...rest
 }) => {
   const context = useControlBarContext();
 
+  const IconButtonWithToolTip = useMemo(
+    () => WithTooltip(IconButton, tooltipContainerId),
+  [tooltipContainerId]);
+
   const ButtonComponent = !!rest['data-tooltip']
     ? IconButtonWithToolTip
     : IconButton;
   const buttonComponentProps = !!rest['data-tooltip-position']
-    ? { tooltipPosition: rest['data-tooltip-position'] }
+    ? { tooltipPosition: rest['data-tooltip-position'], tooltipText }
```

Comment:
Yeah, that's intentional. Some of these components aren't visually displaying text, so they can specify whatever label they want and I don't imagine most builders will run into issues with it

If we get feedback that this isn't the case, we can add them pretty easily. But if it's unecessary, I'd like to avoid it because of this scenario

RosterHeader renders a few components

- `searchLabel` prop - [here](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/src/components/ui/Roster/RosterHeader.tsx#L150)
- an accessibility label - [here](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/src/components/ui/Roster/RosterHeader.tsx#L160)
- `closeLabel` - [here](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/src/components/ui/Roster/RosterHeader.tsx#L167)

Each of these labels would require an additional `labelTooltipText` prop, which just adds to the confusion. So if builders don't run into issues relying on the label field because they aren't rendered visually, I think we should wait

Reply:
Thanks for the thorough explanation :)